docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -96,14 +96,14 @@ function ProductsSwitcher({
   const isBelowMd = useMediaQuery((theme: Theme) => theme.breakpoints.down('md'));
   const productElements = [
     <ProductItem
-      aria-label="Go to core components page"
+      label=", go to core components page"
```

Comment:
```suggestion
      label="by going to the Core components page"
```

Shouldn't it be something like this? It's a sentence following the "Learn more" copy, right? 
> "Learn more by going to the Core components page".